### DISCUSSION TITLE
Fallback to os build-in command when nushell command can't operate due to unknow flag ?

### DISCUSSION THREAD

**Discussion by sandwoodK** [nushell/nushell#6681] [Ideas]
`>ls -ltr`  
would basically fallback to `>^ls -ltr`  instead of raising an error. 

Interactively it would be more accomodant for end-users. Programmatically it should not be a problem as  data type would be (most of the type ? ) different, so programmers are likely to easily spot the mistake due to a type error

**Comment by fdncred** [comment]
I don't really see how this could work. There's nothing to inform nushell the amount of columns or the datatype of the data for each column. For just strings you can try `^ls -ltr | detect columns -s 1 -n` or `^ls -ltr | from ssv -n -a -m 2` or you could use the `parse` command to interpret columns as strings.

**Reply by sandwoodK** [reply]
I am not suggesting such advance behaviour. Only suggesting that in case a nushell command fails , nushell could trigger the native os command if any and return result as plain string.

**Reply by fdncred** [reply]
Seems to be like that would be too surprising, especially if someone mis-types some parameter.
